tests/cpp/operator/mkldnn.cc

```diff
@@ -650,172 +703,135 @@ TEST(MKLDNN_NDArray, CopyFrom) {
       MKLDNNStream::Get()->Submit();
       std::vector<NDArray *> inputs(1);
       inputs[0] = &in_arr.arr;
-      VerifyCopyResult(inputs, out_arr.arr);
+      VerifyCopyResult(inputs, {&out_arr.arr});
     }
   }
 }
 
-void TestUnaryOp(const OpAttrs &attrs, InitFunc init_fn, VerifyFunc verify_fn) {
-  std::vector<NDArray*> inputs(1);
-  std::vector<NDArray*> outputs(1);
-  std::vector<OpReqType> req(1);
-  std::vector<DispatchMode> dispatches = attrs.dispatches;
-
+TEST(MKLDNN_BASE, MKLDNNSum) {
```

Comment:
could you move this test back to the end of the file? It's kind of hard for me to tell what you changed in the test.